lib/modules/pageNavigator.js

```diff
@@ -39,20 +45,32 @@ module.afterLoad = () => {
 	if (module.options.toTop.value) {
 		backToTop();
 	}
+	if (module.options.toComment.value) {
```

Comment:
can I just use the selector `.usertext-edit textarea` to check for existence of the comment area? I don't know of a better way using existing utility functions